app/src/main/res/values/strings.xml

```diff
@@ -101,4 +101,13 @@
 
     <!-- Device dashboard screen -->
     <string name="device_settings">Device settings</string>
+    <string name="temperature">Temperature</string>
+    <string name="humidity">Humidity</string>
+    <string name="temperature_icon">Temperature icon</string>
```

Comment:
Perhaps it was meant to be used in DashboardVAlueBlock? If so, the word "icon" is not needed here. The content description should sound like a title